packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -18,9 +18,12 @@ export enum TabsComponent {
   nav = 'nav'
 }
 
+type ChildElement = React.ReactElement<TabProps, React.JSXElementConstructor<TabProps>>;
+type Child = ChildElement | boolean | null | undefined;
+
 export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivElement>, 'onSelect'>, OUIAProps {
-  /** Content rendered inside the tabs component. Must be React.ReactElement<TabProps>[] */
-  children: React.ReactNode;
+  /** Content rendered inside the tabs component. */
+  children: Child | Child[];
```

Comment:
The way our docs are generated from the props interface, the website is not doing much to describe what the definition of `Child` is. ([see here](https://patternfly-react-pr-6767.surge.sh/components/tabs/#tabs))
The easiest way to fix this for now would probably be to be more descriptive about the type def of `Child` in the prop description comment.

Reply:
@nicolethoen I've pushed a commit to clarify this a bit WDYT?